libs/ledgerjs/packages/types-live/src/feature.ts

```diff
@@ -299,6 +299,11 @@ export type Feature_NewsfeedPage = Feature<{
 
 export type Feature_ProtectServicesMobile = Feature<{
   deeplink: string;
+  compatibleDevices: Array<{
+    available: boolean;
+    comingSoon: boolean;
+    name: string;
+  }>;
   onboardingRestore: {
```

Comment:
Maybe we can use a type for `compatibleDevices` & `onboardingRestore` so we can reuse them, wdyt?

Reply:
Depends on the team sending us the infos. Is this a format they would like to keep and reuse for other purposes ?

Reply:
I only suggest adding types so I'm sure to get your point.
``` 
  type CompatibleDevice = {
    available: boolean;
    comingSoon: boolean;
    name: string;
  };
  ...
  compatibleDevices: CompatibleDevice[];
  ```